wgpu-native/src/device.rs

```diff
@@ -40,8 +40,8 @@ pub extern "C" fn wgpu_device_create_bind_group_layout(
     desc: binding_model::BindGroupLayoutDescriptor,
 ) -> BindGroupLayoutId {
     let bindings = unsafe { slice::from_raw_parts(desc.bindings, desc.bindings_length) };
-    let device = registry::DEVICE_REGISTRY.get_mut(device_id);
-    let descriptor_set_layout = device.device.create_descriptor_set_layout(
+    let device = &mut registry::DEVICE_REGISTRY.get_mut(device_id).device;
```

Comment:
why do we need a mutable reference here?

Reply:
I removed `get` in #10 to match the `&mut` type signature required by parking_lot at the time. I think we should be able to reintroduce `get` now though.